devel-common/src/tests_common/pytest_plugin.py

```diff
@@ -42,14 +42,13 @@
     from itsdangerous import URLSafeSerializer
     from sqlalchemy.orm import Session
 
-    from airflow.models.baseoperator import BaseOperator
     from airflow.models.dag import DAG, ScheduleArg
     from airflow.models.dagrun import DagRun, DagRunType
     from airflow.models.taskinstance import TaskInstance
     from airflow.providers.standard.operators.empty import EmptyOperator
     from airflow.sdk import Context
     from airflow.sdk.api.datamodels._generated import TaskInstanceState as TIState
-    from airflow.sdk.bases.operator import BaseOperator as TaskSDKBaseOperator
+    from airflow.sdk.bases.operator import BaseOperator, BaseOperator as TaskSDKBaseOperator
```

Comment:
This looks odd -- why are we importing it twice under different names?

Reply:
Yeah my editor did that, strange. Made the right change